music21/musicxml/m21ToXml.py

```diff
@@ -4403,12 +4450,32 @@ def noteheadToXml(self, n: note.NotRest) -> Element:
             mxNotehead.set('color', color)
         return mxNotehead
 
+    @staticmethod
+    def topAndBottomIndices(chordOrNote) -> t.Tuple[int, int]:
+        # Use sortAscending if you can, assume ascending if you can't.
+        # If chordOrNote is a note (not expected, but possible), (0, 0) will be returned.
+        # It would be nice if PercussionChord could sortAscending (ascending on the
+        # staff, that is)...
+        notes = list(chordOrNote.notes)
+        top: int = len(notes)
```

Comment:
Chord.notes is a tuple, so I don't think you need to listify it.
```suggestion
        top: int = len(chordOrNote.notes)
```

Reply:
Yes.